lib/zstd.h

```diff
@@ -1109,6 +1109,7 @@ ZSTDLIB_API size_t ZSTD_sizeof_DDict(const ZSTD_DDict* ddict);
 #define ZSTD_FRAMEHEADERSIZE_MIN(format)    ((format) == ZSTD_f_zstd1 ? 6 : 2)
 #define ZSTD_FRAMEHEADERSIZE_MAX   18   /* can be useful for static allocation */
 #define ZSTD_SKIPPABLEHEADERSIZE    8
+#define ZSTD_LITBUFFEREXTRASIZE    1024 /* extra buffer reduces amount of dst required to store litBuffer */
```

Comment:
In which case does this value have to be public ?

It looks to me an implementation detail, that should therefore remain private.
And since it matters only for the decoder, it should be in `lib/decompress`.
Typically `zstd_decompress_internal.h` if it has to be know across multiple decompression units.